SplunkRumWorkspace/SplunkRum/SplunkRum/Session.swift

```diff
@@ -19,7 +19,7 @@ import Foundation
 
 let MAX_SESSION_AGE_SECONDS = 4 * 60 * 60
 
-private var rumSessionId = generateNewSessionId()
+var rumSessionId = generateNewSessionId()
```

Comment:
Please don't make this non-private.  Access to it is controlled by the `sessionIdLock` which you have bypassed in your  `invalidateSession`.